<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Emotion</class>
 <widget class="QWidget" name="Emotion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>146</width>
    <height>109</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>201</width>
     <height>129</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:white;</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em1.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em2.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em3.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>0</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em4.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>37</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em5.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>37</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em6.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>37</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em7.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>37</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em8.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>74</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em9.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>37</x>
      <y>74</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em10.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>74</x>
      <y>74</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em11.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>111</x>
      <y>74</y>
      <width>35</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
     background:url(:/emotion/emo/em12.jpg)
}
QLabel:hover{
    border:1px solid rgb(97, 160, 255);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
